mozjs/src/rust.rs

```diff
@@ -293,6 +294,10 @@ pub struct Runtime {
     /// to represent the resulting ownership graph and risk destroying a Runtime on
     /// the wrong thread.
     outstanding_children: Arc<()>,
+    /// An `Option` that holds the same pointer as `cx`.
+    /// This is shared with all [`ThreadSafeJSContext`]s, so
+    /// they can detect when its destroyed on the main thread.
```

Comment:
Apologies. I made a typo in my suggestion:

```suggestion
    /// they can detect when it's destroyed on the main thread.
```